org.eclipse.mylyn-target/mylyn-e4.26.target

```diff
@@ -134,6 +110,27 @@
 					<version>6.1.0.202203080745-r</version>
 					<type>jar</type>
 				</dependency>
+<!-- https://mvnrepository.com/artifact/org.apache.lucene/lucene-core -->
+<dependency>
+    <groupId>org.apache.lucene</groupId>
+    <artifactId>lucene-core</artifactId>
+    <version>9.5.0</version>
```

Comment:
I wouldn't overtake what the Platform uses, i.e. 9.4.2 https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/blob/master/eclipse.platform.releng.prereqs.sdk/eclipse-sdk-prereqs.target#L32

Reply:
Let me work more on this, it could require both target change and IP review.

Reply:
> Let me work more on this, it could require both target change and IP review.

OK